logger/tests/integration_tests.rs

```diff
@@ -199,10 +301,27 @@ mod needs_docker {
 
         exec_psql(&format!(r#"CREATE DATABASE "{}";"#, &db_name));
 
+        let governor_config = GovernorConfigBuilder::default()
+            .per_millisecond(500)
+            .burst_size(6)
+            .use_headers()
+            .key_extractor(TonicPeerIpKeyExtractor)
+            .finish()
+            .unwrap();
```

Comment:
In theory we could make changes to the main code (rate limiter) and not catch it in the test because of this repeat. Maybe there should be a helper function in the main code that returns the governor? Or something that adds it to the server?

Reply:
Ah, yes, I tried to extract the service builder into a utility function, but I couldn't get it to compile with the complicated types. I tried again now to just extract the config creation into a util, but that was also difficult due to some private types. I will add constants for these values, it's not the cleanest solution, but it should do the trick.